bin/_test-run.sh

```diff
@@ -121,7 +121,7 @@ check_linkerd_binary(){
         exit 1
     fi
     exit_code=0
-    "$linkerd_path" version --client > /dev/null 2>&1
+    "$linkerd_path" version --client 2>&1
```

Comment:
Maybe it's not necessary to output stuff here given we do `exit_on_err` right next, and so we stay in line with`check_if_k8s_reachable`